scripts/databricks_install.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env python
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# This script installs Recommenders/reco_utils as a library into Databricks
```

Comment:
Very minor point, to be more precise, it would be good to add `egg` library into Databricks.

Reply:
updated